src/main.cc

```diff
@@ -74,8 +74,8 @@ it::Record & Application::get_input_type() {
 
 
 
-Application::Application( int argc,  char ** argv)
-: ApplicationBase(argc, argv),
+Application::Application( char ** argv)
```

Comment:
No need for argv.

Reply:
We can't remove it, argv[0] is used as argument of PythonLoader::initialize method.

Reply:
OK so just pass argv[0], that's the program path.